src/styles/_catalog.scss

```diff
@@ -14,9 +14,22 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    cursor: pointer;
+
+    transition: $translation-time ease-in-out;
```

Comment:
Always add transition property. By default, transition property is all, and it's hard for the browser to control all properties.